source/_integrations/thread.markdown

```diff
@@ -201,6 +201,21 @@ Note: the steps and images here show the process with a Google Thread network. B
    ![image](/images/integrations/thread/thread-ha-preferred.png)
    - 🎉 You successfully created a Home Assistant Thread network and joined it with a pre-existing third-party network.
 
+## Migrating to a new Thread adapter
+
+If you already have a Thread adapter running but want to switch to a new one, follow these steps:
+
+1. In your Home Assistant hub, plug in the new adapter.
+2. Add the integration for your new adapter.
```

Comment:
note: So, only ZBT-1/ZBT-2 (and Yellow) really have hardware integrations that can be used for automatically setting up Thread. Others have to be set up manually (firmware flashing + OTBR add-on).

It might be good to explicitly mention this is for Home Assistant Connect devices (+ Yellow?)?

Reply:
hm, this would make this entire procedure specific to Home Assistant Connect? In that case this entire topic should be moved to the support website?